behave/features/environment.py

```diff
@@ -1,26 +1,30 @@
 import os
 import time
+import logging
+import subprocess
 from behave.contrib.scenario_autoretry import patch_scenario_with_autoretry
 from behave.log_capture import capture
+
 from helper.constants import (
     BROWSER,
-    ARTIFACTS_DIRECTORY,
-    DOWNLOAD_DIRECTORY,
+    DATA_DIRECTORY,
     A11Y_TAG,
+    DATABASE_SNAPSHOT_ENABLED,
 )
+
 from helper.helper_web import get_browser
 from features.steps.common_steps import check_accessibility, make_dir, get_tag
-import logging
 
 
 def before_all(context):
+    context.feature_errors_dir = os.path.join(DATA_DIRECTORY, "feature_errors")
```

Comment:
You will need to do this for the A11y report in the `check_accessbility` method in `common_steps.py` as it's set up to use a different file path.